addons/mail/models/res_users_settings.py

```diff
@@ -41,6 +42,20 @@ def _res_users_settings_format(self, fields_to_format=None):
             res['volume_settings_ids'] = [('insert', volume_settings)]
         return res
 
+    def get_backend_only_field_names(self):
+        return set()
+
+    def get_unwanted_field_names(self):
```

Comment:
separate method used only once doesn't look very useful, there is even more doc than code